frames/src/test/java/com/checkout/frames/cvvcomponent/viewmodel/CvvComponentViewModelTest.kt

```diff
@@ -0,0 +1,137 @@
+package com.checkout.frames.cvvcomponent.viewmodel
+
+import android.annotation.SuppressLint
+import androidx.compose.ui.text.input.KeyboardType
+import com.checkout.base.mapper.Mapper
+import com.checkout.base.model.CardScheme
+import com.checkout.frames.cvvcomponent.models.CVVComponentConfig
+import com.checkout.frames.mapper.ContainerStyleToModifierMapper
+import com.checkout.frames.mapper.TextLabelStyleToViewStyleMapper
+import com.checkout.frames.mapper.ImageStyleToComposableImageMapper
+import com.checkout.frames.mapper.InputFieldStyleToViewStyleMapper
+import com.checkout.frames.mapper.InputFieldStyleToInputFieldStateMapper
+import com.checkout.frames.style.component.base.InputFieldStyle
+import com.checkout.frames.style.view.InputFieldViewStyle
+import com.checkout.frames.view.InputFieldState
+import com.checkout.validation.api.CardValidator
+import io.mockk.impl.annotations.RelaxedMockK
+import io.mockk.impl.annotations.SpyK
+import io.mockk.junit5.MockKExtension
+import io.mockk.verify
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.StandardTestDispatcher
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.setMain
+import org.amshove.kluent.internal.assertEquals
+import org.junit.After
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+
+@ExperimentalCoroutinesApi
+@SuppressLint("NewApi")
+@ExtendWith(MockKExtension::class)
+internal class CvvComponentViewModelTest {
```

Comment:
let's add the test cases for:

1. the `isCVVVaild` is valid when a valid CVV is input
1. the `isCVVVaild` is invalid when an invalid CVV is input
1. the `isCVVVaild` is valid when an invalid CVV is input then an valid CVV is input
1. the `isCVVVaild` is invalid when a valid CVV is input then an invalid CVV is input